### DISCUSSION TITLE
[FR] Indicate individual GPX tracks

### DISCUSSION THREAD

**Discussion by yantar92** [minad/osm#51] [Ideas]
Currently, individual GPX tracks are invisible when looking at zoomed out map and indistiniguishable when zooming in.

May it be possible to indicate every GPX track displayed with something similar to POI?

**Comment by minad** [comment]
(Feature request moved to an ideas discussion)

Sounds good. I also miss a way to distinguish tracks. I may implement this when I find time to hack on OSM again. If someone is interested, patches are welcome of course.

**Comment by minad** [comment]
Done in https://github.com/minad/osm/commit/1225a9ef1ad0294ee88dca29d5acbece183fa2bf
